Snowdog_SolrNavigation/templates/grid.phtml

```diff
@@ -0,0 +1,145 @@
+                            href="<?= /* @escapeNotVerified */ $block->getProductUrl($_product) ?>"
+                            class="catalog-grid-item__link"
+                            title="<?= /* @escapeNotVerified */ $_product['name']; ?>"
+                        >
+                            <?= /* @escapeNotVerified */ $_product['name']; ?>
+                        </a>
+                    </h2>
+                    <div class="catalog-grid-item__price">
+                        <div class="price">
+                            <span class="price__value">
+                                <?= /* @escapeNotVerified */ $solrDataHelper->getPrice($_product) ?>
+                            </span>
+                        </div>
+                    </div>
+                </section>
+                <section class="catalog-grid-item__details">
+                    <div
+                        class="
+                            catalog-grid-item__name
+                            catalog-grid-item__name--one-line
+                        "
+                    >
+                        <a
+                            href="<?= /* @escapeNotVerified */ $block->getProductUrl($_product) ?>"
+                            class="catalog-grid-item__link"
+                        >
+                            <?= /* @escapeNotVerified */ $_product['name']; ?>
+                        </a>
+                    </div>
+                    <div class="rating catalog-grid-item__rating">
+                        <div class="rating__star">
+                            <span class="rating__indicator"></span>
+                        </div>
+                    </div>
+                    <div class="catalog-grid-item__price">
+                        <div class="price">
+                            <span class="price__value">
+                                <?= /* @escapeNotVerified */ $solrDataHelper->getPrice($_product) ?>
+                            </span>
+                        </div>
+                    </div>
+                    <div class="catalog-grid-item__actions">
+                        <form
+                            class="catalog-grid-item__primary-form"
+                            action="<?= $block->escapeHtml($block->getAddToUrl('cart', $_product['entity_id'])); ?>"
+                            method="post"
+                            data-mage-init='{ "validation": {} }'
+                        >
+                            <button
+                                class="
+                                    button
+                                    button--secondary
+                                    catalog-grid-item__primary-action
+                                "
+                                type="submit"
+                            >
+                                <span class="catalog-grid-item__cart-text">
+                                    <?= /* @escapeNotVerified */ __('Add to Cart'); ?>
```

Comment:
sometimes there is semicolon at the end of echo sometimes there is not. Please keep the same rule, I propose to remove semicolons

Reply:
Done